test/Mono.Linker.Tests.Cases/RequiresCapability/RequiresUnreferencedCodeCapability.cs

```diff
@@ -163,14 +168,61 @@ static void RequiresUnreferencedCodeOnlyThroughReflection ()
 		{
 		}
 
-		// issue: https://github.com/mono/linker/issues/1607
-		// Linker currently doesn't report a warning in this case
-		// [ExpectedWarning ("IL2026", "--RequiresUnreferencedCodeOnlyThroughReflection--")]
+		[ExpectedWarning ("IL2026", "--RequiresUnreferencedCodeOnlyThroughReflection--")]
 		static void TestRequiresUnreferencedCodeOnlyThroughReflection ()
 		{
-			typeof (RequiresUnreferencedCodeAttribute)
+			typeof (RequiresUnreferencedCodeCapability)
 				.GetMethod (nameof (RequiresUnreferencedCodeOnlyThroughReflection), System.Reflection.BindingFlags.Static | System.Reflection.BindingFlags.NonPublic)
 				.Invoke (null, new object[0]);
 		}
+
+		class BaseType
+		{
+			[RequiresUnreferencedCode ("Message for --VirtualMethodRequiresUnreferencedCode--")]
+			public virtual void VirtualMethodRequiresUnreferencedCode ()
+			{
+			}
+		}
+
+		class TypeWhichOverridesMethod : BaseType
+		{
+			public override void VirtualMethodRequiresUnreferencedCode ()
```

Comment:
This should be currently producing a warning - all overrides must have the same annotations.